SIL.Windows.Forms/ClearShare/MetaData.cs

```diff
@@ -361,7 +362,7 @@ public bool FileFormatSupportsMetadata(string path)
 			return file != null && !file.GetType().FullName.Contains("NoMetadata");
 		}
 
-		public void Write(string path)
+		public void Write(string path, string originalImagePath = null)
```

Comment:
> For example, if we apply the same ClearShare.metadata to every file in a bloom book, doing that would make them all appear to have been taken in the same place at the same time, etc.

In that scenario, we either need to filter what gets into the examplar to just Intellectual Property, or when we apply it, limit the application to IP.